src/impac-angular.module.js

```diff
@@ -121,14 +122,18 @@ angular.module('impac.components.dashboard-settings',
 );
 angular.module('impac.components.widgets-common',
   [
-    'impac.components.widgets-common.data-not-found',
     'impac.components.widgets-common.editable-title',
     'impac.components.widgets-common.info-panel',
     'impac.components.widgets-common.time-period-info',
     'impac.components.widgets-common.top-buttons',
     'impac.components.widgets-common.autofocus'
   ]
 );
+angular.module('impac.components.common-settings',
+  [
+    'impac.components.common-settings.data-not-found'
```

Comment:
Can you review the naming? "data not found" state is not a setting per say... maybe just "common" ?